numpy/distutils/intelccompiler.py

```diff
@@ -54,34 +55,38 @@ def __init__(self, verbose=0, dry_run=0, force=0):
                              linker_so=compiler + ' -shared')
 
 
-class IntelCCompilerW(MSVCCompiler):
-    """
-    A modified Intel compiler on Windows compatible with an MSVC-built Python.
-    """
-    compiler_type = 'intelw'
-
-    def __init__(self, verbose=0, dry_run=0, force=0):
-        MSVCCompiler.__init__(self, verbose, dry_run, force)
-        version_match = simple_version_match(start='Intel\(R\).*?32,')
-        self.__version = version_match
-
-    def initialize(self, plat_name=None):
-        MSVCCompiler.initialize(self, plat_name)
-        self.cc = self.find_exe("icl.exe")
-        self.lib = self.find_exe("xilib")
-        self.linker = self.find_exe("xilink")
-        self.compile_options = ['/nologo', '/O3', '/MD', '/W3', '/Qstd=c99']
-        self.compile_options_debug = ['/nologo', '/Od', '/MDd', '/W3',
+if sys.platform == 'win32':
+    from distutils.msvc9compiler import MSVCCompiler
+    class IntelCCompilerW(MSVCCompiler):
```

Comment:
Charris, thanks for letting me know this. I just tried the pep8 online checker and correct serveral coding styles errors.